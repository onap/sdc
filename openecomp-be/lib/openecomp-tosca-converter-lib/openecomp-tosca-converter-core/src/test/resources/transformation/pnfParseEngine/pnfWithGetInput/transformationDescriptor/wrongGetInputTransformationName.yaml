- transformation-for: nodeTemplate
  name: PnfExtCp transformation
  description: converts any node template that has a type tosca.nodes.nfv.PnfExtCp
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query:
        properties:
          trunk_mode:
      to-name: trunk_mode_copy
      to-value:
        strategy: copy
    - query:
        properties:
          role:
      to-name: role
      to-value:
        strategy: copy
      to-get-input: stringTransformation1
- transformation-for: getInputFunction
  name: stringTransformation
  description: converts a string input
  conversions:
    - query:
        type:
        invalid-another-attribute: #cannot have two attributes in the attribute query
      to-name: type
      to-value:
        strategy: copy
    - query:
        description:
      to-name: description
      to-value:
        strategy: copy
    - query:
        default:
      to-name: default
      to-value:
        strategy: copy