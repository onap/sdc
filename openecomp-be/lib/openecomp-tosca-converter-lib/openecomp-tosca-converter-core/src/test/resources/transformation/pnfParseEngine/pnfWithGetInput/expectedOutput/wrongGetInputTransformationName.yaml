topology_template:
  node_templates:
    pnfExtCp_3:
      properties:
        trunk_mode_copy: false
        role: {get_input: role1}
    pnfExtCp_2:
      properties:
        trunk_mode_copy: false
        role: {get_input: role}
    pnfExtCp_1:
      properties:
        trunk_mode_copy: false
        role: leaf